<?xml version="1.0"?>
<launch>
	
	<!-- Run Gmapping node-->
	<include file="$(find tp_base)/launch/gmapping.launch"/>
	
	<!-- Run the move_base node-->
	<include file="$(find tp_base)/launch/move_base.launch"/>
	
	<!--Run Kinect and depthimage_to_laserscan node-->
	<include file="$(find tp_base)/launch/sensor_source.launch"/>
	
	<!-- Run the serial server_ odometry node-->
	<include file = "$(find tp_serial)/launch/tp_serial.launch" />
	
	<!-- Run the sensor transforms node-->
	<node pkg = "tp_base" type = "robot_tf_publisher" name = "robot_tf_publisher" output = "screen">
		<param name = "tilt_angle" value = "0.0" type = "double" />
		<param name = "camera_height" value = "0.5" type = "double" />
	</node>
	
</launch>
